tests/integration/cronenbroguelike/commands_test.py

```diff
@@ -0,0 +1,135 @@
+import io
+import unittest
+from unittest.mock import call
+from unittest.mock import Mock
+from unittest.mock import patch
+
+import adventurelib
+
+from cronenbroguelike import ability
+from cronenbroguelike import commands
```

Comment:
This isn't used, please remove.

Reply:
It is implicitly. Importing commands registers all the `@when`-decorated functions; without this line, the whole house of cards would come down.

Reply:
Can you leave a comment to that effect in the code? This is unintuitive/unexpected.

Reply:
Yes, and I added a pragma so your linter doesn't complain.